src/test/java/org/openstreetmap/atlas/geography/atlas/command/AtlasFindByAtlasIdentifierSubCommandTest.java

```diff
@@ -0,0 +1,54 @@
+package org.openstreetmap.atlas.geography.atlas.command;
+
+import java.io.PrintStream;
+import java.util.Arrays;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openstreetmap.atlas.streaming.resource.File;
+
+/**
+ * Unit test for {@link AtlasFindByAtlasIdentifierSubCommand}.
+ *
+ * @author bbreithaupt
+ */
+public class AtlasFindByAtlasIdentifierSubCommandTest
+{
+    private final String shardPath = AtlasFindByAtlasIdentifierSubCommandTest.class
+            .getResource("DNK_Copenhagen").getPath();
+
+    @Test
+    public void testConsoleOutput()
+    {
+        // Redirect System.out to capture AtlasFindByFeatureIdentifierLocatorSubCommand output
+        final PrintStream originalOut = System.out;
+        final CaptureOutputStream captureStream = new CaptureOutputStream(originalOut);
+        System.setOut(captureStream);
+
+        // Run AtlasFindByAtlasIdentifierSubCommand
+        final String[] args = { "find-atlas-id", String.format("-input=%1$s", shardPath),
+                "-id=546649246000001,575954012000000" };
+        new AtlasReader(args).runWithoutQuitting(args);
+
+        // Reset System.out
+        System.setOut(originalOut);
```

Comment:
Any chance something goes wrong on lines 29-31 and we never reset System.out? Possible solution could be a try/catch/finally block.